### PR TITLE
pizza pizza

### PR DESCRIPTION
**Pull request by ShizCalev**
:cl: ShizCalev
fix: Pineapple haters/lovers will get/no longer get pineapple pizzas respectively from infinite pizza boxes.
/:cl:

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
what

**Comment by HomocidlePancakes** [comment]
PIZZA PASTA PEPERONI LA LA LA LA SPAFETTI!

**Comment by Nickkoranda** [comment]
Based and pizzapilled

File: code/modules/food_and_drinks/pizzabox.dm [hunk 1]
```diff
@@ -329,7 +329,13 @@
 /obj/item/pizzabox/infinite/proc/attune_pizza(mob/living/carbon/human/noms) //tonight on "proc names I never thought I'd type"
 	if(!pizza_preferences[noms.ckey])
 		pizza_preferences[noms.ckey] = pickweight(pizza_types)
-		if(noms.mind && noms.mind.assigned_role == "Botanist")
+		if(noms.has_quirk(/datum/quirk/pineapple_liker))
+			pizza_preferences[noms.ckey] = /obj/item/reagent_containers/food/snacks/pizza/pineapple
+		else if(noms.has_quirk(/datum/quirk/pineapple_hater))
+			var/list/pineapple_pizza_liker = pizza_types.Copy()
+			pineapple_pizza_liker -= /obj/item/reagent_containers/food/snacks/pizza/pineapple
+			pizza_preferences[noms.ckey] = pineapple_pizza_liker
```
Comment (AnturK) [inline code/modules/food_and_drinks/pizzabox.dm]: Missing pick here.

**Review by AnturK — CHANGES_REQUESTED** [review]

**Review by AnturK — APPROVED** [review]

**Pull request merged**